packages/components/src/font-size-picker/utils.ts

```diff
@@ -44,3 +44,20 @@ export function getCommonSizeUnit( fontSizes: FontSize[] ) {
 	} );
 	return areAllSizesSameUnit ? firstUnit : null;
 }
+
+/**
+ * hasValidShortName
+ *
+ * check that the font size short name is a string with at most
+ * 3 characters length (e.g. "XXS", "XS", "S", "M", "L", "XL", "XXL")
+ *
+ * @param  shortName font size object
+ * @return boolean
+ */
+export function hasValidShortName( shortName: string | undefined ): boolean {
```

Comment:
Either remove the `| undefined` here or remove the `fontSize.shortName &&` check above. It's unnecessary to have both.

Reply:
I was not able to make TypeScript happy when removing the `fontSize.shortName &&` portion so I removed the `| undefined` type from the util in [b33e8c4](https://github.com/WordPress/gutenberg/pull/46586/commits/b33e8c4233f88b996b94d4c68c084676772f6b22)